pom.xml

```diff
@@ -32,8 +33,9 @@
         <jenkins.version>1.625.3</jenkins.version>
         <java.level>7</java.level>
         <jenkins-test-harness.version>2.13</jenkins-test-harness.version>
-        
+
         <azuresdk.version>1.3.0</azuresdk.version>
+        <azure-commons.version>0.2.4-SNAPSHOT</azure-commons.version>
```

Comment:
update to release version:

Reply:
Updated after azure-commons-0.2.4 release